maro/rl/policy_v2/policy_base.py

```diff
@@ -212,22 +213,23 @@ def get_batch_loss(self, batch: dict, explicit_grad: bool = False) -> object:  #
         """
         pass
 
-    @abstractmethod
     def data_parallel(self, *args, **kwargs) -> None:
         """"Initialize a proxy in the policy, for data-parallel training.
         Using the same arguments as `Proxy`."""
-        pass
+        self.task_queue_client = TaskQueueClient()
```

Comment:
1. Instance attribute `task_queue_client` defined outside `__init__ `
2. Should we use `_task_queue_client` if it is an internal variable?

Reply:
The `TaskQueueClient` will be created only in the `data_parallel` mode, and another method `data_parallel_with_existing_proxy` met the same problem.